README.rst

```diff
@@ -100,6 +100,28 @@ For an actively developed codes for PDF simulations see the
 DiffPy-CMI framework at http://www.diffpy.org.
 
 
+INSTALLATION AND PACKAGE BUILDING VERIFICATION
+-----------------------------------------------
+To verify pdffit2 installed correctly and that the package is buildable across OS’s from Conda-forge, Github, and PyPi use the following procedure:
+- Create a new conda env on your local machine and preset the python version to match python that the current version pdffit2 is running on.
+   - Conda-forge: **Recommended**, follow the steps above for the installation process.
+   - Github: Download the tar file for the newest version under Release Tab on the GitHub page, then run the setup file (same command as above)
+   ::
+      python setup.py install
```

Comment:
its usually recommended to run `pip install -e .` from the directory that contains setup.py rather than the command you wrote.  It effectively does the same thing, but has some slightly different behavior that I don't understand that is more friendly somehow.